tools/testrunner/local/pool.py

```diff
@@ -6,14 +6,17 @@
 # for py2/py3 compatibility
 from __future__ import print_function
 
-from Queue import Empty
 from contextlib import contextmanager
-from multiprocessing import Process, Queue
```

Comment:
As I understand it, we can use `multiprocessing` here, as it is compatible with both python 2 and python 3.

case with:
```
-from multiprocessing import Process, Queue
+try:
+  from queue import Empty  # Python 3
+except ImportError:
+  from Queue import Empty  # Python 2
```

gives error

```
ERROR: setUpClass (__main__.SystemTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "unittests/run_tests_test.py", line 155, in setUpClass
    pool.setup_testing()
  File "/opt/node/tmp/v8/tools/testrunner/local/pool.py", line 29, in setup_testing
    del Queue
NameError: global name 'Queue' is not defined
```